server/packaging/server/in-docker-build.sh

```diff
@@ -7,18 +7,33 @@ sed -i -e '\;bindings;d' Cargo.toml
 
 # Install Rust (actual toolchain is going to be installed by maturin according to `rust-toolchain.toml`)
 curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain none
-
 export PATH="/root/.cargo/bin:$PATH"
+cargo --version
+
+# Install Poetry
+curl -sSL https://install.python-poetry.org | python
```

Comment:
Could you consider fixing the installed poetry version ?

Reply:
good idea ! done.